### PR TITLE
NanoUI Upgrades (Part 1) [Gone Wrong][Gone Sexual][Gone Insane]

### LINKED ISSUE #2998: Nuke interface is hilariously broken
hilarity

Thanks, everyone who *completely neglected to mention this to me*

### PR DESCRIPTION
**Pull request by tigercat2000**
Rediddilydo of #2865 

Changes Nano to use one script file, compilation via Gulp
Starts a README.md for nano
Drives Tiger Insane

Nah, but seriously, I got it working correctly this time, don't ask me how to compile it, because it's documented in the README.md and I double-tested it every which way I could think, it works.

Shit to do next PR:
 - LESS/SASS CSS stuff
 - Maaaybe port FontAwesome

Also removed some useless shit, like "Copy to BYOND cache" and the debug.html.
Fixes #2998

### CODE REVIEW

File: nano/scripts/libraries/0_jquery.js [hunk 1]
```diff
@@ -8513,7 +8513,7 @@ var rvalidbraces = /(?:^|:|,)(?:\s*\[)+/g;
 var rvalidescape = /\\(?:["\\\/bfnrt]|u[\da-fA-F]{4})/g;
 var rvalidtokens = /"[^"\\\r\n]*"|true|false|null|-?(?:\d+\.|)\d+(?:[eE][+-]?\d+|)/g;
 
-jQuery.parseJSON: function( data ) {
+jQuery.parseJSON = function( data ) {
```
Comment (tigercat2000) [inline nano/scripts/libraries/0_jquery.js]: I have absolutely no fucking clue how this was working, because `:` is not in any way valid here

**Comment by tigercat2000** [comment]
Oh I forgot
nuke

**Comment by tigercat2000** [comment]
Oh, I should probably make TRAVIS test the scripts.... or something...
whatever don't care not touching it IT WORKS MROWL

**Comment by Taaketa** [comment]
10,000,000 views for that title on youtube.

**Comment by Markolie** [comment]
:+1: x :100:

With regards to the nuke interface: My only suggestion would be to move the time (114) in "Set Time" out of the label and next to the buttons (the content) so it looks better and the text is white.

**Comment by tigercat2000** [comment]
@Markolie I'd love to do that, but text does not play well with generated links, at all.

**Comment by Markolie** [comment]
I think you can do:
<div style="float: left;">114</div>{{:helper.link('--', '', null, 'disabled')}}

**Comment by tigercat2000** [comment]
@Markolie Alright, I'll play around with it for a while, see if I can make something decent looking.

**Comment by tigercat2000** [comment]
@Markolie Does this look alright?
mrowl

I managed this with some random CSS3 fiddling, apparently inline-flex works for what I want

**Comment by Markolie** [comment]
Lookin' good.

**Comment by tigercat2000** [comment]
I just had an idea, does this look better or worse than the above, @Markolie 
less hacky 2

**Comment by tigercat2000** [comment]
Went with this color scheme:
active
disabled

**Comment by Plotron** [comment]
Outstanding.

**Comment by CrAzYPiLoT-SS13** [comment]
I think the first one was better, but oh, well.

**Comment by Iamgoofball** [comment]
its just a prank bro

**Comment by tigercat2000** [comment]
👌👀👌👀👌👀👌👀👌👀 good shit go౦ԁ sHit👌 thats ✔ some good👌👌shit right👌👌th 👌 ere👌👌👌 right✔there ✔✔if i do ƽaү so my selｆ 💯  i say so 💯  thats what im talking about right there right there (chorus: ʳᶦᵍʰᵗ ᵗʰᵉʳᵉ) mMMMMᎷМ💯 👌👌 👌НO0ОଠＯOOＯOОଠଠOoooᵒᵒᵒᵒᵒᵒᵒᵒᵒ👌 👌👌 👌 💯 👌 👀 👀 👀 👌👌Good shit

**Comment by Tastyfish** [comment]
[These threads are getting far too silly](https://raw.githubusercontent.com/ParadiseSS13/Paradise/master/sound/voice/shriek1.ogg)

**Pull request merged**